tests/server/googleLogin.test.js

```diff
@@ -0,0 +1,89 @@
+import { insertUser } from "../../server/api/googleLogin";
+import { db } from "../../server/db";
+import * as Crypto from "crypto";
+import Supertest from "supertest";
+
+const App = require("../../server/index");
+const request = Supertest(App);
+
+jest.mock("../../server/db");
+
+describe("Test insertUser", () => {
```

Comment:
add a `beforeAll` that mocks `console.log` and `console.err`

i think all the other tests should have the code